python/private/common/common.bzl

```diff
@@ -15,6 +15,7 @@
 
 load(":cc_helper.bzl", "cc_helper")
 load(":providers.bzl", "PyInfo")
+load("//python/private:reexports.bzl", "BuiltinPyInfo")
```

Comment:
Looks like the tests are failing due to this line.

Reply:
Fixed. The exports.bzl dep was missing.